Cargo.toml

```diff
@@ -13,3 +14,16 @@ debug = 0
 [profile.release]
 lto = "thin"
 codegen-units = 16
+
+[workspace.dependencies]
+emily = { version = "0.1.0", path = ".generated-sources/emily" }
+tokio = "1.32.0"
+serde = "1.0"
+serde_json = "1.0"
+lambda_runtime = "0.11.1"
+aws_lambda_events = "0.15.0"
+http = "1.1.0"
+aws-sdk-dynamodb = { version = "1.3.0" }
+futures = "0.3.24"
+# This is necessary to compile the AWS Lambda as a lambda.
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Should I be updating these here, or on a package basis?

Reply:
I'm okay with either, but I think workspace root makes the most sense. 

As an aside, are you sure that openssl is necessary for AWS Lambda? As in, why wouldn't rustls work? I think this dependency is fine, since having it vendored makes potential compilation issues go away (I think), I just recall having issues with openssl when building with MUSL (might have been my fault, but still).

Reply:
This bit is predictive of the next PR, I can remove it for this one. It's not great practice to include things needed for the next PR.

This is what it's needed for:
- #103 

As part of that PR we need to run this command to compile the `.zip` file that a is unpacked and run in the lambda. 

```sh
cargo lambda build --release --output-format zip [--arm64]
```
The [Cargo Lambda](https://www.cargo-lambda.info/) command uses shared object files during compilation, and MacOS uses the sources in the directory marked by the environment variable `OPENSSL_DIR`. The problem is mac uses files like `libcrypto.3.dylib` instead of `libcrypto.3.so` which is what the compiler needs. 

After looking through a bunch of solutions this was the only thing that seemed to work across platforms. I also wonder if there was a better solution but this looks to be the standard one.